tutorials/clustering.md

```diff
@@ -13,48 +12,53 @@ In this tutorial, we will show how you can create a distributed hypertable withi
+- The superuser role, `postgres` in this example, must have an entry in the `pg_hba.conf` file for the data nodes. This should look something like: ```host    all    postgres    access.example.com    trust``` You can use any CIDR block covering the access node's IP in place of `access.example.com`. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication].
+
+After making any of these changes, please restart your Postgres instance to ensure the
+new settings take effect.
 
 ## Define the topology for your distributed database
 
-Now that you have 3 running instances of TimescaleDB, you can go ahead and set up your
-distributed database. Connect to the instance that you would like to act as the access node,
-and create a new database there:
+Now that you have 3 running instances of TimescaleDB, you can go ahead
+and set up your distributed database. Connect to `access.example.com`
+using your superuser role, and create a new database there. It's
+important that none of the instances already have a `multinode`
+database on them before starting this step:
 
 ```sql
 CREATE DATABASE multinode;
-\c multinode
+\c multinode postgres
 CREATE EXTENSION timescaledb;
 ```
 
 Now, you can tell your access node that currently contains the multinode database to add
 data nodes in order to store distributed hypertables:
 
 ```sql
-SELECT add_data_node('dn1',host=>'<address>', if_not_exists=>'true',
-password=>'<remote_password>', bootstrap_user=>'<superuser>',
-bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('dn1', host=>'data1.example.com');
 
-SELECT add_data_node('dn2',host=>'<address>', if_not_exists=>'true',
-password=>'<remote_password>', bootstrap_user=>'<superuser>',
-bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('dn2', host=>'data2.example.com');
 ```
 
-Let's go through exactly what the `add_data_node` command is doing. The first
-argument represents the local name of a data node that you can use to refer to
-data nodes when running commands on the access node. The `host` parameter is
-what the access node will use to actually access the remote data node. The
-`if_not_exists` parameter will bootstrap a database on the remote data node if
-an associated database does not yet exist. The `password` parameter is used each
-time the access node connects to a data node once the data node is created. This
-`password` is associated with the current user of the system. The
-`bootstrap_user` and `bootstrap_password` parameters are used when a data node is
-initially logged into, and needs to be a superuser. If you're already logged in as a
-superuser, the `bootstrap_user` and `bootstrap_password` parameters are optional.
+The first argument represents the local name of a data node that you can
+use to refer to data nodes when running commands on the access node. The
+`host` parameter is the IP or DNS the access node will use to actually
```

Comment:
Suggest "the IP or DNS" ==> "the address of"

Reply:
went with "IP or DNS address of the data node."  I wanted to make it clear how to do this if the user is doing everything by IP.

Reply:
It is not very clear what a DNS address is. Is it the address of the DNS server or something else? I think you should call it domain name.